polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -71,6 +72,18 @@ $nav-max-width: 360px;
     align-items: center;
     height: $top-bar-height;
     padding: 0 var(--p-space-2) 0 var(--p-space-4);
+    background-color: var(--p-surface);
+    box-shadow: var(--p-shadow-top-bar);
+    margin-bottom: var(--p-space-4);
+
+    &::after {
+      content: '';
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      border-bottom: var(--p-border-width-1) solid transparent;
+    }
```

Comment:
Why does this need to be positioned like this? Could we move this to a parent element if it's trying the break a bounding area? Does the border-bottom work after the `margin-bottom`?

Reply:
That was an oversight. I removed it.